doc/usage.rst

```diff
@@ -97,29 +97,34 @@ directory:
     Entering directory '/home/jdimino/code/dune'
     ...
 
-More precisely, it will choose the outermost ancestor directory containing a
-``dune-workspace`` file as root. For instance if you are in
-``/home/me/code/myproject/src``, then dune will look for all these files in
-order:
-
--  ``/dune-workspace``
--  ``/home/dune-workspace``
--  ``/home/me/dune-workspace``
--  ``/home/me/code/dune-workspace``
--  ``/home/me/code/myproject/dune-workspace``
--  ``/home/me/code/myproject/src/dune-workspace``
-
-The first entry to match in this list will determine the root. In
-practice this means that if you nest your workspaces, dune will
-always use the outermost one.
-
-In addition to determining the root, ``dune`` will read this file
-to setup the configuration of the workspace unless the ``--workspace``
-command line option is used. See the section :ref:`dune-workspace`
-for the syntax of this file.
-
-The ``Entering directory`` message can be suppressed with the
-``--no-print-directory`` command line option (as in GNU make).
+This message can be suppressed with the ``--no-print-directory``
+command line option (as in GNU make).
+
+Dune requires at least one ``dune-workspace`` file or ``dune-project``
+file to be present in the current directory or a parent one in order
+to operate. Such a file is used to mark the root of the current
+workspace.
+
+However, they are treated slightly differently. Since there can be
+only a single workspace active at a given time, Dune stops its search
+for the root at the first ``dune-workspace`` file it encounters.
+
+On the other hand, Dune projects are composable and there can be
+multiple projects in a single workspace. For this reason, when Dune
+finds a ``dune-project`` file it will continue searching in parent
+directories in case this project is part of a bigger workspace.
+
+A ``dune-workspace`` file is also a configuration file. ``dune`` will
```

Comment:
Should this be "Dune will read it unless the ``--workspace`` command line option is used" or is ``dune`` correct because it's a command or specific process? I guess I'm asking if it's "Dune" the tool/program or a specific ``dune`` command.

Reply:
I was thinking of `dune` the command. But thinking about it more, talking about `dune` the command is more relevant when talking about running Dune from the command lines. What we are describing here applies however the user runs Dune, so Dune might be better. Changing this.